### DISCUSSION TITLE
Assess the impact of filtering the current data on estimates

### DISCUSSION THREAD

**Discussion by ben18785** [pints-team/pints#1534] [Ideas/Science]
Test different filtering methods from basic thinning to more sophisticated filters.

**Comment by MichaelClerx** [comment]
Saw one example today where very noisy data created bias in the log-likelihood estimate! Got less when using more samples (understandably) so that'll be one thing to look out for

**Comment by MichaelClerx** [comment]
These guys don't like it 
http://onlinelibrary.wiley.com/doi/10.1111/j.2041-210X.2011.00131.x/abstract
blog post about this paper:
http://doingbayesiandataanalysis.blogspot.co.uk/2011/11/thinning-to-reduce-autocorrelation.html

**Comment by mirams** [comment]
Thinning the data and thinning the MCMC chains are quite different things ;-)

**Comment by ben18785** [comment]
Yep, thinning chains is used to reduce memory footprint.

**Comment by mirams** [comment]
or to hide badly mixing ones?!?

**Comment by MichaelClerx** [comment]
Woops!

**Comment by MichaelClerx** [comment]
We do thinning in the current MCMC examples, should we remove that?
It's also a default option for the adaptive covariance mcmc routine

**Comment by ben18785** [comment]
I wouldn't make it a default option, but think we want to retain the functionality for memory footprint issues.

**Comment by MichaelClerx** [comment]
Hey @ben18785 , seems to me we're doing this one (kind of) in e.g. Richard's problem. Should we maybe close this as out of scope? Unless we can turn it into a solid feature request for PINTS
